seeds/events.js

```diff
@@ -27,6 +27,7 @@ exports.seed = function(knex, Promise) {
             location: event.location.longitude + ',' + event.location.latitude,
             cover_image: event.coverImage,
             fb_event_id: event.facebookId,
+            radius: event.radius || process.env.EVENT_RADIUS,
```

Comment:
I think the env var could be named `DEFAULT_EVENT_RADIUS` to emphasise it's the default value, not necessarily the only value